OMEdit/OMEdit/OMEditGUI/Options/OptionsDialog.cpp

```diff
@@ -3661,6 +3669,10 @@ SimulationPage::SimulationPage(OptionsDialog *pOptionsDialog)
   mpIgnoreCommandLineOptionsAnnotationCheckBox = new QCheckBox(tr("Ignore __OpenModelica_commandLineOptions annotation"));
   // ignore simulation flags annotation checkbox
   mpIgnoreSimulationFlagsAnnotationCheckBox = new QCheckBox(tr("Ignore __OpenModelica_simulationFlags annotation"));
+  //! @todo Remove this once we enable this bydefault in OMC.
+  /* Enable new instantiation use in OMC API */
+  mpEnableNewInstantiationAPICheckBox = new QCheckBox(tr("Enable experimental new instantiation use in the OMC API"));
```

Comment:
I would suggest "Enable new front end use in the OMC API (faster GUI response)". We always referred to "new front end" when marketing this thing, I'm afraid "new instantiation" would be confusing. Also, this is no longer supposed to be experimental, once beta-tested, the aim of 1.14.0 is to have this thing production-grade